src/containers/Tag/TagAdd/TagCheckbox.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from "react";
+import {
+    Checkbox,
+} from "@material-ui/core";
+
+import styles from "./TagCheckbox.module.css";
+
+interface TagCheckboxProps {
+    name: string,
+    value: boolean,
+    callBack: Function,
+}
+
+export const TagCheckbox: React.SFC<TagCheckboxProps> = (props: TagCheckboxProps) => {
```

Comment:
This is good abstraction. However, we should move this into an independent component which is generic and not specific to tag.  Similar to /src/components/UI/Button.